src/internal/m365/service/onedrive/mock/handlers.go

```diff
@@ -22,6 +24,8 @@ import (
 type BackupHandler struct {
 	ItemInfo details.ItemInfo
 
+	EDID EnumeratesDriveItemsDelta
```

Comment:
please use more meaningful names for variables. EDID gives no context as to what the variable actually holds